CHANGELOG.md

```diff
@@ -71,7 +71,7 @@
 
 ### Version 1.5.8 ([Maven Central](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.netflix.hystrix%22%20AND%20v%3A%221.5.8%22), [Bintray](https://bintray.com/netflixoss/maven/Hystrix/1.5.8/)) ###
 
-* [Pull 1419](https://github.com/Netflix/Hystrix/pull/1419) When user has not opted in to letting core/maximum threadpools diverge, ensure dynamic updates to coreSize apply to both
+* [Pull 1419](https://github.com/Netflix/Hystrix/pull/1419) When user has not opted into letting core/maximum threadpools diverge, ensure dynamic updates to coreSize apply to both
```

Comment:
`in to` -> `into` changes have been quite controversial.

Happy to drop this.

---

Some projects don't like changing changelogs...